src/styles/page-template/index.md.njk

```diff
@@ -157,6 +157,14 @@ To change the components that are included in the page template by default, set
       </td>
     </tr>
 
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">cspNonce</td>
+      <td class="govuk-table__cell">Variable</td>
+      <td class="govuk-table__cell">
+        Add a `nonce` attribute to the script for your Content Security Policy (CSP). Provide a nonce that hostile actors cannot guess, as otherwise they can easily find a way around your CSP. However, you should use this attribute only if you're not able to [include the hash for the inline scripts in your service's CSP](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#if-your-javascript-is-not-working-properly).
```

Comment:
I'm not sure why, but Markdown links don't work in this context 🤷 

Oh, and the apostrophe should probably be curly?

```suggestion
        Add a `nonce` attribute to the script for your Content Security Policy (CSP). Provide a nonce that hostile actors cannot guess, as otherwise they can easily find a way around your CSP. However, you should use this attribute only if you're not able to <a class="govuk-link" href="https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#if-your-javascript-is-not-working-properly">include the hash for the inline scripts in your service’s CSP</a>.
```

Reply:
@lfdebrux Thanks! Is that the apostrophe around nonce in "nonce attribute"?

Reply:
Ah, I was referring to the possessive apostrophe on `service's`. I hadn't noticed the quote marks around ``nonce``, I wonder if that needs the `<code>` markup...

Reply:
Ah gotcha! Yeah, that makes sense re apostrophe.

For the other suggestion, is this what we want?

```suggestion
        Add a <code>nonce</code> attribute to the script for your Content Security Policy (CSP). Provide a nonce that hostile actors cannot guess, as otherwise they can easily find a way around your CSP. However, you should use this attribute only if you're not able to <a class="govuk-link" href="https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#if-your-javascript-is-not-working-properly">include the hash for the inline scripts in your service’s CSP</a>.
```

Reply:
Yep, that's great :)

Reply:
Thank you!